os/pm/pm_procfs.c

```diff
@@ -203,6 +203,8 @@ static void power_read_domains(void (*readprint)(const char *, ...))
 static void power_read_state(void (*readprint)(const char *, ...))
 {
 	enum pm_state_e pm_state;
+
+	readprint("PM %s\n\n", (g_pmglobals.is_running) ? "RUNNING" : "STOPPED");
```

Comment:
procfs is not only for information.
sometimes application should parse data from procfs, so it shoul memcpy data to buffer.
Hence if you want reveal running state, then you should create additional directory, not here.